services/transition/docker-compose.yml

```diff
@@ -0,0 +1,51 @@
+version: '3.7'
+
+x-transition: &transition
+  build:
+    context: .
+    dockerfile: Dockerfile.govuk-base
+  image: transition
+  volumes:
+    - ${GOVUK_ROOT_DIR:-~/govuk}:/govuk:delegated
+    - home:/home/build
+  working_dir: /govuk/transition
+
+services:
+  transition-lite:
```

Comment:
We should be able to get the cucumber tests passing with the `privileged` flag.

https://github.com/alphagov/govuk-docker/blob/master/services/content-publisher/docker-compose.yml#L16

Reply:
At the moment, we've just got https://github.com/alphagov/govuk-developer-docs/pull/2081/files#diff-d21cd4fd2ed7b37c176270e625ad3014R258